src/main/resources/archetype-resources/silo/pom.xml

```diff
@@ -17,31 +17,29 @@
         <tag>HEAD</tag>
     </scm>
     <properties>
-        <HikariCP.version>2.4.3</HikariCP.version>
+        <HikariCP.version>2.5.0</HikariCP.version>
         <aspectjweaver.version>1.8.7</aspectjweaver.version>
         <flyway-core.version>4.0.3</flyway-core.version>
-        <gelf4j.version>1.9</gelf4j.version>
         <h2.version>1.4.190</h2.version>
         <hamcrest-all.version>1.3</hamcrest-all.version>
-        <jackson-databind.version>2.6.4</jackson-databind.version>
-        <javax.servlet-api.version>3.1.0</javax.servlet-api.version>
-        <json-path.version>2.1.0</json-path.version>
+        <jackson-databind.version>2.8.2</jackson-databind.version>
+        <json-path.version>2.2.0</json-path.version>
         <junit.version>4.12</junit.version>
-        <maven-apt-plugin.version>1.0.4</maven-apt-plugin.version>
+        <apt-maven-plugin.version>1.1.3</apt-maven-plugin.version>
```

Comment:
Are we aware of the fact, that we cannot use Classes with it, that have AspectJ on board? 

See https://github.com/querydsl/apt-maven-plugin for the information.